src/releases/mod.rs

```diff
@@ -129,23 +153,59 @@ impl<C: Clone + Connect> Releases<C> {
         format!("/repos/{}/{}/releases{}", self.owner, self.repo, more)
     }
 
+    /// Create new a release.
+    ///
+    /// See the [github docs](https://developer.github.com/v3/repos/releases/#create-a-release)
+    /// for more information.
     pub fn create(&self, rel: &ReleaseOptions) -> Future<Release> {
         self.github.post(&self.path(""), json!(rel))
     }
 
+    /// Edit a release by id.
+    ///
+    /// See the [github docs](https://developer.github.com/v3/repos/releases/#edit-a-release)
+    /// for more information.
     pub fn edit(&self, id: u64, rel: &ReleaseOptions) -> Future<Release> {
         self.github
             .patch(&self.path(&format!("/{}", id)), json!(rel))
     }
 
+    /// Delete a release by id.
+    ///
+    /// See the [github docs](https://developer.github.com/v3/repos/releases/#delete-a-release)
+    /// for more information.
     pub fn delete(&self, id: u64) -> Future<()> {
         self.github.delete(&self.path(&format!("/{}", id)))
     }
 
+    /// List published releases and draft releases for users with push access.
+    ///
+    /// See the [github docs](https://developer.github.com/v3/repos/releases/#list-releases-for-a-repository)
+    /// for more information.
     pub fn list(&self) -> Future<Vec<Release>> {
         self.github.get(&self.path(""))
     }
 
+    /// Return the latest full release. Draft releases and prereleases are not returned.
+    ///
+    /// See the [github docs](https://developer.github.com/v3/repos/releases/#get-the-latest-release)
+    /// for more information.
+    pub fn get_latest(&self) -> Future<Release> {
```

Comment:
I don't have super strong opinions about this but for rustyness sake there are some suggestions in the rust style guide that suggest avoiding [get_ prefixes](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html#getter/setter-methods-[rfc-344])

If you don't feel strongly about holding onto this name, what do you think about just `latest` and something like `by_tag` `for_tag` or `with_tag` below?

Reply:
i actually thought about these variants for a moment after i opened the PR